contrib/checkstyle/test/src/mill/contrib/checkstyle/CheckstyleModuleTest.scala

```diff
@@ -0,0 +1,67 @@
+package mill
+package contrib.checkstyle
+
+import mill.scalalib.JavaModule
+import mill.testkit.UnitTester
+import mill.testkit.TestBaseModule
+import utest._
+
+object CheckstyleModuleTest extends TestSuite {
+
+  val res = os.Path(sys.env("MILL_TEST_RESOURCE_FOLDER"))
+  val ver = "10.18.1"
+
+  def tests = Tests {
+    val src = "sbt-checkstyle"
+
+    test("checkstyle") {
+      test("plain") {
+        cs("plain", "6.2") - src
+        cs("plain", "6.3") + src
+        cs("plain") + src
```

Comment:
I have no idea how to read your short names here. Please make them more verbose and do not use unnecessary operators